lib/backend_common/default.nix

```diff
@@ -128,6 +128,7 @@ in {
     echo "## pytest ######################################################"
     echo "################################################################"
     pytest tests/ -vvv
+    # pytest tests/ -vvv -s --log-level=DEBUG
```

Comment:
Why?

Reply:
when debugging failed tests i needed it. not needed now at all

Reply:
Oh sorry, for some reason I saw you commented out the pytest invocation, I didn't notice it was just a commented-out verbose invocation :P